testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestingIcebergStorageFormat.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.iceberg;
+
+import org.testng.annotations.DataProvider;
+
+import java.util.stream.Stream;
+
+public enum TestingIcebergStorageFormat
+{
+    PARQUET,
+    ORC,
+    // not supported in Trino
+    AVRO,
```

Comment:
Style suggestion, maybe specify Trino support using a constructor argument, like `AVRO(false)` ?

Reply:
thought about that, but that would be much more code to write. i chose simpler version